schedule/schedule.go

```diff
@@ -87,6 +87,15 @@ type Shift struct {
 	StopDate time.Time `json:"stopDate,omitempty"`
 }
 
+// StartDateExclusive returns the date before the start date, which is typically the StopDateInclusive of the previuos
```

Comment:
Why is the `StartDateExclusive` only "typically" the `StopDateInclusive` of the previous shift?

Reply:
You're right, this is just me hedging unnecessarily.